src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceOfStyles = sourceString
+    .split(';')
+    .map(param => param
+      .split(':')
+    );
+  const result = {};
+
+  for (const param of sourceOfStyles) {
+    if (param[1]) {
```

Comment:
logic is understandable = > if you got the second element then you have first by default, but better to check the existence of both of them